backend/sns/users/service.py

```diff
@@ -802,6 +802,31 @@ def get_followings(
 
         return followings
 
+    def read_is_followed(
+        self,
+        db: Session,
+        current_user_id: int,
+        checked_user_id: int,
+    ) -> bool:
+        """현재 로그인한 유저와 다른 유저 간 팔로우 관계가 맺어져 있는지 확인한다.
+
+        Args:
+
+        - current_user_id (int): 현재 로그인한 유저의 id
+        - checked_user_id (int): 확인받는 유저의 id
+
+        Returns:
+
+        - bool: 팔로우 관계가 맺어져 있으면 True, 아니면 False를 반환
+        """
+        follow: Follow = user_crud.get_follow(
+            db,
+            follower_id=checked_user_id,
+            following_id=current_user_id,
+        )
+
+        return follow.is_followed if follow else False
```

Comment:
follow.is_followed 자체가 불리언이니 그냥 return 해도 될 것 같습니다. 

```suggestion
        return follow.is_followed
```

Reply:
@K021 get_follow가 one_or_none 메서드를 사용하기 때문에 None을 반환할 경우 follow.is_followed 를 사용할 경우 None에는 is_followed 속성이 없다는 에러가 발생될 수 있어서 위와 같이 작성했습니다.

Reply:
아 맞네요!